sofria_mediaId.js

```diff
@@ -1,44 +1,144 @@
+if (process.argv.length < 2) {
+    throw new Error(`usage: sofria-mediaId <USX FILESET DIRECTORY>`);
+}
+
+const PARENT_PATH = './output';
+
+const crypto = require('crypto').webcrypto;
 const path = require('path');
 const fse = require('fs-extra');
-const {Proskomma} = require('proskomma');
+const {
+    Proskomma
+} = require('proskomma');
+const cliProgress = require('cli-progress');
 
-// CLI: sofria-mediaId <USX FILESET DIRECTORY>
-// example: sofria-mediaId testdata/ENGESVN_ET-usx
-// result: create a corresponding sofria json fileset, eg:
-//  1. create testdata/ENGESVN_ET-json/ directory (note that it is adjacent to the USX FILESET DIRECTORY)
-//  2. populate it with single-chapter sofria json files (created from the input usx files)
+if (global.crypto !== 'object') {
+    global.crypto = {
+        getRandomValues: (array) => crypto.getRandomValues(array)
+    };
+}
 
-if (process.argv.length !== 5) {
-    throw new Error(`usage: sofria-mediaId <USX FILESET DIRECTORY>`);
+const progressBar = new cliProgress.SingleBar({}, cliProgress.Presets.shades_classic);
```

Comment:
note that the main use case for sofria-cli will be headless (eg invoked from dbp-etl, and therefore no UI). It's ok to have the progressBar capability as long as it works in a headless environment

Reply:
ok I got it. It has been removed.